pytorch_lightning/core/lightning.py

```diff
@@ -146,6 +149,78 @@ def forward(self, x):
         if self.trainer.is_global_zero:
             print(*args, **kwargs)
 
+    def log(
```

Comment:
```suggestion
    def track(
```
would it be better name which dos not colide with loggers...